src/components/SocialButton.js

```diff
@@ -40,14 +40,14 @@ export const SocialButton = props => {
 
   return (
     <Touchable
-      rippleColor="black"
+      rippleColor={theme.colors.background.grey800}
       disabled={disabled}
       bg="transparent"
       height={48}
       width="100%"
       borderRadius={8}
       borderWidth={1}
-      borderColor={theme.colors.border.black}
+      borderColor={theme.colors.background.grey800}
```

Comment:
We can fetch the same from `theme.colors.border.grey800` instead of `background`.  Although both will set the same color, in this since we are setting the border color that should ideally be fetch from the `border` object right.